pkg/services/authz/rbac/mapper.go

```diff
@@ -101,13 +112,67 @@ func newResourceTranslation(resource string, attribute string, folderSupport, sk
 	}
 }
 
+// newDashboardTranslation creates a translation for dashboards and also maps the actions to action sets
+func newDashboardTranslation(resource string, attribute string, folderSupport, skipScopeOnCreate bool) translation {
+	dashTranslation := newResourceTranslation(resource, attribute, folderSupport, skipScopeOnCreate)
```

Comment:
Nit. We don't need them as parameters. Same for folders.
```suggestion
func newDashboardTranslation() translation {
	dashTranslation := newResourceTranslation("dashboards", "uid", true, false)
```